tempodb/backend/gcs/config.go

```diff
@@ -1,12 +1,19 @@
 package gcs
 
-import "time"
+import (
+	"time"
+
+	"cloud.google.com/go/storage"
+)
 
 type Config struct {
-	BucketName        string        `yaml:"bucket_name"`
-	ChunkBufferSize   int           `yaml:"chunk_buffer_size"`
-	Endpoint          string        `yaml:"endpoint"`
-	Insecure          bool          `yaml:"insecure"`
-	HedgeRequestsAt   time.Duration `yaml:"hedge_requests_at"`
-	HedgeRequestsUpTo int           `yaml:"hedge_requests_up_to"`
+	BucketName string `yaml:"bucket_name"`
```

Comment:
i prefer the ordering/organization of this struct before.

Reply:
This update was to try an satisfy the maligned struct lint check, but wasn't complete enough to fix it.  Still in progress.